core/cap-0045.md

```diff
@@ -334,34 +334,37 @@ Y' = Y .
 ```
 
 We will now show that the liquidity pool does take the actions described above.
-If `p < (1 - F) Y/X` then
-```
-X' = (pX + Y) (1 - F) / (2 - F) / p
-   = X (1 - F) / (2 - F) + Y (1 - F) / (2 - F) / p
-   > X (1 - F) / (2 - F) + Y (1 - F) / (2 - F) / ((1 - F) Y / X)
-   = X (1 - F) / (2 - F) + X / (2 - F)
-   = X
-Y' = (pX + Y) / (2 - F)
-   = pX / (2 - F) + Y / (2 - F)
-   < (1 - F) Y / (2 - F) + Y / (2 - F)
-   = Y
-```
-as expected based on the description that the liquidity pool buys `X` and sells
-`Y`. If `1/p < (1 - F) X/Y` then
-```
-X' = (pX + Y) / (2 - F) / p
-   = X / (2 - F) + Y / (2 - F) / p
-   < X / (2 - F) + Y / (2 - F) * ((1 - F) X / Y)
-   = X / (2 - F) + X (1 - F) / (2 - F)
-   = X
-Y' = (pX + Y) (1 - F) / (2 - F)
-   = pX (1 - F) / (2 - F) + Y (1 - F) / (2 - F)
-   > X (1 - F) / (2 - F) / ((1 - F) X / Y) + Y (1 - F) / (2 - F)
-   = Y / (2 - F) + Y (1 - F) / (2 - F)
-   = Y
-```
-as expected based on the description that the liquidity pool sells `X` and buys
-`Y`.
+First, suppose that `p < (1 - F) Y/X`.  
+Then the liquidity pool sells `Y` and buys `X` until
+`p = (1 - F) Y'/X'`.  Combining this equation with the budget constraint gives
+`pX + Y = Y' + (1 - F)Y'`.
+Rearranging gives `Y' = (pX + Y) / (2 - F)` and `X' = (pX + Y) (1 - F) / (p * (2 - F))`.
+
+Observe that in this case, 
+`X' = (pX + Y) (1 - F) / (p * (2 - F))
```

Comment:
Can we change these 4 inline code segments to be out-of-line so they are easy to read? Based on how you wrote them, I feel like that is what you meant to do anyway.

Reply:
By "these 4" I mean the 4 in the two "Observe that..." blocks.

Reply:
Makes sense.  Yeah, that's what I intended to do.